projects/documentation/content/deprecation.md

```diff
@@ -0,0 +1,108 @@
+6. **_Migration Period_**: Deprecated components will remain functional for a specified period to allow users to migrate to alternative solutions.
+7. **_Removal:_** Deprecated components will be removed from the library in the 1.0.0 release. Prior to removal, the deprecation level will be changed to red to indicate the urgency of migration.
+
+## Examples
+
+### Deprecating a function, method or class
+
+Its important functions, methods and classes are never immediately removed from code.
+Instead they are deprecated for at least 1 major release before being removed.
+
+A function,method or a class can be depreacted with a depreacted tag in the JSdoc to properly surface up to its context and API
+
+```js
+*
+* @deprecated since 0.42.1
+*/
+
+const enableScrollIntoView = () => {
+    ....
+}
+```
+
+### Adding Deprecation Notice
+
+Context:
+
+-   This example demonstrates how a deprecation notice is added to a component using the `window.__swc.warn` method.
+-   The `window.__swc.warn` method is used to display a warning message to developers using the deprecated component.
+
+Purpose:
+
+-   The purpose of this deprecation notice is to inform developers using the deprecated component about its status and encourage them to migrate to alternative solutions in dev mode.
+-   The notice provides a link to documentation for more information on migrating to alternative solutions.
+
+```js
+window.__swc.warn(
+    this,
+    `<${this.localName}> is deprecated. Use a Button Group to show any additional actions related to the most critical action.`,
+    'https://opensource.adobe.com/spectrum-web-components/components/split-button/#deprecation',
+    { level: 'deprecation' }
+);
+```
+
+### Adding `package.json` update
+
+Context:
+
+-   This example demonstrates how the `package.json` file of a deprecated component is updated with a `deprecationNotice` key.
+-   The `deprecationNotice` key contains a message informing users about the deprecation and providing guidance on alternative solutions.
+
+Purpose:
+
+-   The purpose of updating the `package.json` file with a `deprecationNotice` key is to provide a clear and visible indication to users that the component is deprecated.
+-   This helps users to easily identify the deprecated component and access guidance on migrating to alternative solutions.
+
+```json
+{
+    "name": "@spectrum-web-components/split-button",
+    "version": "0.0.1",
+    "deprecationNotice": "@spectrum-web-components/split-button is deprecated."
+}
+```
+
+## Backward compatibility
+
+Breaking changes in Spectrum Web Components will only happen in the platform releases (ie: major version releases). Any planned breaking changes will be communicated with advance notice through the deprecation method and documentation.
+
+Breaking changes will be deprecated at least 2 feature releases or six months ahead (whatever is smaller).
```

Comment:
This seems to conflict with our earlier statements about changes deprecated during 1.0 being removed in 2.0. I think this should be a little more clear what to expect?

I'm happy saying something like:

> deprecations will be removed at the next major release, if possible. If the specifics for a deprecation can't be constrained to the next major release, we'll minimally mark things as deprecated for 2 releases or six months (whatever is smaller).